### ISSUE TITLE
[REQUEST] Snapchat support

### ISSUE THREAD

**Issue by Walmann** [AAndyProgram/SCrawler#162]
Add support for snapchat.

**Describe the solution you'd like**
Download for private accounts may be hard, but download for public profiles has already been done with python: https://github.com/skyme5/snapchat-dl

**Comment by AAndyProgram** [comment]
No, but you can [create a plugin](https://github.com/AAndyProgram/SCrawler/wiki/Plugins) for SCrawler.
